_config.yml

```diff
@@ -0,0 +1,22 @@
+excludeOnPrivateRepo:
+  # Repos without public users don't need this stuff
+  - LICENSE.md
+  - CONTRIBUTING.md
+  - .github/SUPPORT.md
+  - .github/ISSUE_TEMPLATE/config.yml
+  
+  # Code in private repos isn't generally signed
+  - src/NServiceBus.snk
+  - src/NServiceBusTests.snk
+  
+  # No GitHub release artifacts to scan
+  - .github/workflows/virus-scan.yml
+excludeOnReleaseBranch:
+  # GitHub only reads this info on the default branch
+  - .github/SUPPORT.md
+  - .github/ISSUE_TEMPLATE/config.yml
+  - .github/ISSUE_TEMPLATE/bug_report.yml
+  - .github/ISSUE_TEMPLATE/feature_request.yml
+  - .github/ISSUE_TEMPLATE/improvement_request.yml
+  - .github/workflows/stale.yml
```

Comment:
Do we not want to mark stale PR's on release branches? Or do we think it's so rare as to not warrant the resource to run the workflow (does that matter to us?)?

Reply:
That workflow runs on a cron trigger, so the only version of the file that matters is the version on the default branch.

Reply:
Ah, so the job on the main branch does all PR's, regardless of target branch. Gotcha.